packages/venia-ui/lib/components/Newsletter/newsletter.js

```diff
@@ -0,0 +1,97 @@
+import React, { Fragment, useEffect } from 'react';
```

Comment:
Import order should be changed, and imports within venia-ui should be changed to relative paths.

Example
```
import React, { Fragment, useEffect } from 'react';
import { FormattedMessage, useIntl } from 'react-intl';
import { Form } from 'informed';
import { shape, string } from 'prop-types';

import { useNewsletter } from '@magento/peregrine/lib/talons/Newsletter/useNewsletter';
import { useToasts } from '@magento/peregrine';

import { isRequired } from '../../util/formValidators';
import { mergeClasses } from '../../classify';
import FormError from '../FormError';
import Button from '../Button';
import LoadingIndicator from '../LoadingIndicator';
import TextInput from '../TextInput';
import defaultClasses from './newsletter.css';
```